firebase_admin/ml.py

```diff
@@ -541,16 +543,17 @@ def from_saved_model(cls, saved_model_dir, bucket_name=None, app=None):
         """
         TFLiteGCSModelSource._assert_tf_enabled()
         tflite_model = TFLiteGCSModelSource._tf_convert_from_saved_model(saved_model_dir)
-        open('firebase_ml_model.tflite', 'wb').write(tflite_model)
-        return TFLiteGCSModelSource.from_tflite_model_file(
-            'firebase_ml_model.tflite', bucket_name, app)
+        open(model_file_name, 'wb').write(tflite_model)
```

Comment:
I missed this earlier. But this is leaking a file descriptor. Can you change this to:

```
with open(model_file_name, 'wb) as model_file:
    model_file.write(tflite_model)
```